src/lib/bootloader/autoinst_profile/bootloader_section.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+# Copyright (c) [2020] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "installation/autoinst_profile/section_with_attributes"
+require "bootloader/autoinst_profile/global_section"
+require "bootloader/autoinst_profile/device_map_entry_section"
+
+module Bootloader
+  module AutoinstProfile
+    # This class represents an AutoYaST `<bootloader>` section
+    #
+    class BootloaderSection < ::Installation::AutoinstProfile::SectionWithAttributes
+      def self.attributes
+        [
+          { name: :loader_type },
+          { name: :loader_device }, # deprecated
+          { name: :activate }, # deprecated
+          { name: :sections } # deprecated
+        ]
+      end
+
+      define_attr_accessors
+
+      # @!attribute loader_type
+      #   @return [String] which boot loader to use (default, grub2, grub2-efi and none)
+      #   @see Bootloader::BootloaderFactory::SUPPORTED_BOOTLOADERS
+
+      # @!attribute loader_device
+      #   @deprecated
+
+      # @!attribute activate
+      #   @see GlobalSection#activate
+      #   @deprecated
+
+      # @!attribute sections
+      #   @deprecated
```

Comment:
deprecated and also ignored, as in grub2 we do not support own sections.

Reply:
Yes, that's it. Actually, the method is there just because of [this warning](https://github.com/yast/yast-bootloader/blob/ef272ca255c45c06fc4fbe6a4ec9f1aef564e5d0/src/lib/bootloader/autoyast_converter.rb#L49). If you are fine with dropping the warning, we can drop this code too.

Reply:
I am fine without it. It should just not crash if it is there ( even if it s removed in SLE12 already )